wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/StringEscapeSequence.scala

```diff
@@ -0,0 +1,27 @@
+package wdl.model.draft3.elements
+
+import cats.syntax.validated._
+import common.validation.ErrorOr.ErrorOr
+import wdl.model.draft3.elements.ExpressionElement.{BackslashEscape, DoubleQuoteEscape, NewlineEscape, SingleQuoteEscape, StringEscapeSequence, TabEscape, UnicodeCharacterEscape}
+
+object StringEscapeSequence {
+  val Octal = "\\\\([0-7]{3})".r
+  val Hex = "\\\\x([0-9a-fA-F]{2})".r
+  val FourDigitUnicode = "\\\\u([0-9a-fA-F]{4})".r
+  val EightDigitUnicode = "\\\\U([0-9a-fA-F]{8})".r
+
+  def parseEscapeSequence(seq: String): ErrorOr[StringEscapeSequence] = seq match {
```

Comment:
We test a wider range of escape sequences in `string_escaping.wdl` - will these still work?

Of course, we are allowed to change the behavior, with consensus - I doubt they see much use.

Reply:
I'm not touching the WDL 1.0 parser in this change, which means that any WDL 1.0 string literal runs through the hermes-encapsulated `wdl_unescape` function and becomes a plain old `:string`.

This code will only run over `:escape` tokens. 

And yes, it's deliberate that the set is smaller in Biscayne - per openwdl/wdl#272

Reply:
To be clear I referred to the 1.0 file because it's just a convenient list to point at